.cirrus.yml

```diff
@@ -1,24 +1,74 @@
-task:
-  freebsd_instance:
-    matrix:
-      image: freebsd-12-4-release-amd64
-      image: freebsd-13-1-release-amd64
+setup: &FREEBSD_SETUP
   env:
-    CIRRUS_SHELL: /bin/sh
-    RUST_VERSION: stable
-    matrix:
-      FEATURES: libnv
-      FEATURES: nvpair
-  install_script:
+    HOME: /tmp # cargo needs it
+    RUST_BACKTRACE: full  # Better info for debugging test failures.
+  setup_script:
+    - pkg install -y llvm valgrind
     - fetch https://sh.rustup.rs -o rustup.sh
-    - sh rustup.sh -y --default-toolchain ${RUST_VERSION}
-    - pkg install -y llvm
+    - sh rustup.sh -y --profile=minimal --default-toolchain ${VERSION}
+
+build: &BUILD_TEST
   cargo_cache:
     folder: $HOME/.cargo/registry
+    fingerprint_script: cat Cargo.lock || echo ""
   build_script:
-    - . $HOME/.cargo/env
-    - cargo build --no-default-features --features=$FEATURES
+    - . $HOME/.cargo/env || true
+    - cargo build ${FEATURES}
   test_script:
-    - . $HOME/.cargo/env
-    - cargo test --no-default-features --features=$FEATURES
+    - . $HOME/.cargo/env || true
+    - cargo test ${FEATURES}
+  leak_script:
```

Comment:
You can actually remove this. I just wanted to make sure I didn't miss anything because documentation wasn't clear enough about who is responsible for release. (yay rust ownership semantics)

Reply:
Remove running the leak example?